.env.example

```
@@ -1,16 +1,4 @@
-GITHUB_TOKEN=
-LINEAR_API_KEY=
-NUXT_BETTER_AUTH_SECRET=
-NUXT_GITHUB_CLIENT_ID=
-NUXT_GITHUB_CLIENT_SECRET=
-NUXT_INTELLIGENCE_SECRET=
-VERCEL_OIDC_TOKEN=
-SLACK_BOT_TOKEN=
-TYPEFULLY_API_KEY=
-SHELVE_TOKEN=
\ No newline at end of file
+SHELVE_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SHELVE_TOKEN key should go before the STUDIO_GITHUB_CLIENT_ID key

(UnorderedKey)